### PR TITLE
Citations import - format authors

### PR DESCRIPTION
**Pull request by dglemos**
Format authors to store the first 4 authors. If longer than 4, include 'et al'. 
ENSVAR-4182

### CODE REVIEW

File: scripts/import/import_EPMC.pl [hunk 1]
```diff
@@ -346,9 +346,30 @@ sub get_publication_info_from_epmc{
...
+    ### format authors list
+    my $new_authors = format_authors($ref->{resultList}->{result}->{authorString});
+    $ref->{resultList}->{result}->{authorString} = $new_authors unless !$new_authors;
+
     return $ref;
 }
 
+# Format authors list
+# only store the first 4 authors
+sub format_authors{
```
Comment (at7) [inline scripts/import/import_EPMC.pl:358]: I would be more precise here and call it trim_author_list

File: scripts/import/import_EPMC.pl [hunk 2]
```diff
@@ -346,9 +346,30 @@ sub get_publication_info_from_epmc{
          return undef;
     }
 
+    ### format authors list
+    my $new_authors = format_authors($ref->{resultList}->{result}->{authorString});
```
Comment (at7) [inline scripts/import/import_EPMC.pl:350]: instead of new_authors maybe call it trimmed_author_list

File: scripts/import/import_EPMC.pl [hunk 3]
```diff
@@ -346,9 +346,30 @@ sub get_publication_info_from_epmc{
...
+    $ref->{resultList}->{result}->{authorString} = $new_authors unless !$new_authors;
```
Comment (at7) [inline scripts/import/import_EPMC.pl:351]: I would prefer: 
$ref->{resultList}->{result}->{authorString} = $trimmed_author_list if $trimmed_author_list;

File: scripts/import/import_EPMC.pl [hunk 4]
```diff
@@ -511,6 +532,10 @@ sub check_dbSNP{
             print $error_log "WARN dbSNP data:\t$l->[1]\t($ref->{resultList}->{result}->{title}) - species not mentioned\n"
         }
 
+        # format authors list
+        my $new_authors = format_authors($ref->{resultList}->{result}->{authorString});
```
Comment (at7) [inline scripts/import/import_EPMC.pl:536]: same comments as above on method and variable naming

File: scripts/import/import_EPMC.pl [hunk 5]
```diff
@@ -981,6 +1006,10 @@ sub parse_UCSC_file{
         my ($rs, $pmid, $section, $doi, $title, $authors, $year, $extId ) = split/\t/;
         next if $section =~/refs|ack/; ## not in this publication
 
+        # format authors list
+        my $new_authors = format_authors($authors);
```
Comment (at7) [inline scripts/import/import_EPMC.pl:1010]: same comments as above on method and variable naming

**Comment by dglemos** [comment]
@at7 thanks for the suggestions. I've renamed the variables.

**Review by at7 — APPROVED** [review]

**Comment by at7** [comment]
merged into master and release/105

**Pull request merged**